media/js/firefox/firstrun/experiment-firstrun-email-first.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    /**
+     * An experiment to show FxA's "email-first" flow where
+     * the user first enters their email and then /signin
+     * or /signup is displayed depending on whether the
+     * account exists.
+     *
+     * 5% go to each bucket, 10% total.
+     */
+    var cop = new Mozilla.TrafficCop({
```

Comment:
Do you test TrafficCop files?

Reply:
The traffic cop library is already pretty heavily unit tested, so I think this is fine as-is.